ledger-tool/src/main.rs

```diff
@@ -1425,10 +1425,16 @@ fn main() {
                     .help("Include sysvars too"),
             )
             .arg(
-                Arg::with_name("exclude_account_data")
-                    .long("exclude-account-data")
+                Arg::with_name("no_accounts_contents")
+                    .long("no-accounts-contents")
                     .takes_value(false)
-                    .help("Exclude account data (useful for large number of accounts)"),
+                    .help("Do not print contents of each account, which is very slow with lots of accounts."),
+            )
+            .arg(
+                Arg::with_name("no_account_data")
+                    .long("no-account-data")
+                    .takes_value(false)
+                    .help("Do not print account data when printing account contents."),
```

Comment:
I renamed "exclude account data" to "no account data". Is that OK? I've noticed most of our flags are "no-" to disable bits.

Reply:
I don't think we have `solana-ledger-tool accounts` scripted anywhere, so it should be okay to make these breaking changes.